### PR TITLE
Add vsclust

### PR DESCRIPTION
**Pull request by veitveit**
Describe your pull request here
VSClust is a clustering tools for quantitative omics data
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by veitveit** [comment]
@BiocondaBot please add label

**Comment by veitveit** [comment]
@BiocondaBot please update

File: recipes/vsclust/meta.yaml
Comment (epruesse) [inline recipes/vsclust/meta.yaml:45]: You should not need to change directories into the installation directory.

**Comment by epruesse** [comment]
@Bioconda/r Is the `build.sh` done the Rrrright way?

File: recipes/vsclust/meta.yaml
Comment (veitveit) [reply recipes/vsclust/meta.yaml]: Thanks, I did not know that. This should be solved now.

File: recipes/vsclust/meta.yaml
Comment (epruesse) [inline recipes/vsclust/meta.yaml:54]: This could be wrapped at 72 chars widths, so it can be read in github a little better.

**Review by epruesse — DISMISSED** [review]
Need to fix the location of the files. `..../bin` is for executables only.

**Comment by epruesse** [comment]
Meh. Github deleted most of my comments. Comes down to this: Do not install everything to `/bin`. That's in the $PATH, every package installs their executables there, should not be spammed with many files, as few and as uniquely named as possible, and just the execs please.

You an put data files in $PREFIX/share/PKGNAME or $PREFIX/lib/PKGNAME, or in case of R packages, wherever R prefers it (probably somewhere in $PREFIX/lib/R/....). See also [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard). 

If those extra files are just for the tests, don't add them to the package at all. List them as test files in the meta.yaml and perhaps point the user to where they are in the online manual / docs. People don't usually check those folders for conda packages.

File: recipes/vsclust/meta.yaml
Comment (veitveit) [reply recipes/vsclust/meta.yaml]: @epruesse 
Thanks for the detailed requests. 
Now the bin folder only contains the binaries and I also solved the other issues.

**Comment by veitveit** [comment]
@BiocondaBot please fetch artifacts

**Comment by veitveit** [comment]
Did some tests and they failed, so please do not merge yet

**Comment by veitveit** [comment]
@BiocondaBot please fetch artifacts

**Comment by veitveit** [comment]
All seems to work now.

@BiocondaBot please add label

**Comment by bwlang** [comment]
I'm not sure what is this t file.
I'll leave this to someone who know more about this type of package...

**Comment by veitveit** [comment]
@bwlang I checked the content of https://121730-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages/linux-64/vsclust-0.87-r36h4333106_0.tar.bz2 and could not see any "t file". 

Maybe you looked into an older commit?

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**
